src/styles/card.css

```diff
@@ -0,0 +1,107 @@
+/* card region */
+* {
+  box-sizing: border-box;
+}
+
+.card {
+  width: 200px;
+  padding: 32px 16px 16px;
+  margin: 0;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
```

Comment:
Do you need the flex container in the card ?

Reply:
Yes, without that everything breaks down

Reply:
We don't need flex in the card container